### PR TITLE
SOW-24: wrap form in app - WIP

### PR DESCRIPTION
**Pull request by vferraro-scottlogic**
## Changes
- add App containing Form
- add vite-config for building app
- [x] change back to main branch for build-and-deploy workflow
- [x] uncomment jobs in build-and-deploy workflow
- [x] Update readme

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -3,21 +3,23 @@ name: Build and Deploy
...
     paths:
       [
-        ".github/workflows/storybook.yml",
+        ".github/workflows/deploy.yml",
+        "vite.config.ts",
         "lib/stories/**",
         "lib/**/*",
+        "src/**/*",
         "package-lock.json",
       ]
```
Comment (Ro4052) [inline .github/workflows/deploy.yml:15]: We're probably at the point where we're not really getting any benefit from these paths, maybe we should just get rid of them?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -27,34 +29,34 @@ jobs:
           cache: "npm"
 
       - run: npm ci
-      - run: npm run build:storybook
+      - run: npm run build
 
       - name: Deploy Storybook
```
Comment (Ro4052) [inline .github/workflows/deploy.yml:34]: We should rename this step

File: src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+const App = (): JSX.Element => {
+  return (
+    <>
+      <Form />
+    </>
+  );
+};
+
+export default App;
```
Comment (Ro4052) [inline src/App.tsx:11]: Not sure what your opinion is on them but personally I'm not a fan of default exports

Comment (vferraro-scottlogic) [reply src/App.tsx:11]: I agree. I am in the habit of doing a default export for App but I don't have a good reason for it really. Happy to make it a regular export

File: package.json [hunk 4]
```diff
@@ -9,12 +9,14 @@
     "dist"
   ],
   "scripts": {
+    "dev": "vite",
```
Comment (Ro4052) [inline package.json:12]: If we want, we can have this automatically open in the browser:

```suggestion
    "dev": "vite --open",
```

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (Ro4052) [reply .github/workflows/deploy.yml:15]: That's basically already happening anyway and the files that aren't included here aren't changing very often

**Review by Ro4052 — APPROVED** [review]

**Pull request merged**